src/test/java/org/broadinstitute/hellbender/engine/ReadsDataSourceUnitTest.java

```diff
@@ -573,7 +573,7 @@ public void testManuallySpecifiedIndices( final List<Path> bams, final List<Path
     public void testManuallySpecifiedIndicesWithCustomReaderFactory( final List<Path> bams, final List<Path> indices ) {
         final SamReaderFactory customFactory = SamReaderFactory.makeDefault().validationStringency(ValidationStringency.STRICT);
 
-        try ( final ReadsDataSource readsSource = new ReadsDataSource(bams, indices, customFactory) ) {
+        try ( final ReadsDataSource readsSource = new ReadsDataSource(bams, indices, customFactory, null, null) ) {
```

Comment:
Revert this change, and add a new, separate unit test for the five-arg constructor that passes in dummy wrappers of some kind and assert that they've been applied.